hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app/src/main/java/org/apache/hadoop/mapreduce/v2/app/rm/RMContainerRequestor.java

```diff
@@ -213,12 +213,10 @@ protected AllocateResponse makeRemoteRequest() throws YarnException,
         allocateResponse.getCompletedContainersStatuses().size();
 
     if (ask.size() > 0 || release.size() > 0) {
-      LOG.info("getResources() for " + applicationId + ":" + " ask="
-          + ask.size() + " release= " + release.size() + " newContainers="
-          + allocateResponse.getAllocatedContainers().size()
-          + " finishedContainers=" + numCompletedContainers
-          + " resourcelimit=" + availableResources + " knownNMs="
-          + clusterNmCount);
+      LOG.info("applicationId={} : asking={} release={} newContainers={} finishedContainers={}"
```

Comment:
```suggestion
      LOG.info("applicationId={} : ask={} release={} newContainers={} finishedContainers={}"
```

Reply:
Thanks @szilard-nemeth for review. I have addressed your comment.